projects/ngrx.io/content/guide/store/selectors.md

```diff
@@ -16,18 +16,18 @@ When using the `createSelector` and `createFeatureSelector` functions @ngrx/stor
 import { createSelector } from '@ngrx/store';
 
 export interface FeatureState {
-  counter: number;
+counter: number;
 }
 
 export interface AppState {
-  feature: FeatureState;
+feature: FeatureState;
 }
 
 export const selectFeature = (state: AppState) => state.feature;
 
 export const selectFeatureCount = createSelector(
-  selectFeature,
-  (state: FeatureState) => state.counter
+selectFeature,
+(state: FeatureState) => state.counter
```

Comment:
Hi @briannarenni,

Can you revert all formatting changes from this file and keep only this one:

```diff
-export const selectFeature = createFeatureSelector&lt;AppState, FeatureState&gt;(featureKey);
+export const selectFeature = createFeatureSelector&lt;FeatureState&gt;(featureKey);
```

Reply:
Reverted formatting (forgot to turn off Format on Save for this).